tests/test_computed_fields.py

```diff
@@ -427,7 +427,10 @@ def _double(self) -> int:
     assert m.model_dump() == {'x': 2, '_double': 4}
 
 
-@pytest.mark.skipif(sys.version_info < (3, 9), reason='@computed_field @classmethod @property only works in 3.9+')
+@pytest.mark.skipif(
+    sys.version_info < (3, 9) or sys.version_info >= (3, 13),
+    reason='@computed_field @classmethod @property only works in 3.9-3.12',
```

Comment:
Is there a way to do this in 3.13?

Reply:
I didn't find an easy way - I suppose you could create a custom decorator that basically handles the `classmethod` stuff with `__wrapped__` as suggested here:

https://github.com/python/cpython/pull/92379/files#diff-78f24041d66ab8ed2ae1aee94bcd42396d27af833cb96a3c506294c6d6dce82d

Reply:
Should a new issue be opened for this?